l10n/en-US/viewer.ftl

```diff
@@ -369,6 +369,11 @@ pdfjs-editor-signature-add-signature-button =
     .title = Add new signature
 pdfjs-editor-signature-add-signature-button-label = Add new signature
 
+# Variables:
+#   description (String) - a string describing/labeling the signature.
+pdfjs-editor-add-saved-signature-button =
+    .title = Saved signature: { $description }
```

Comment:
Can you help me understand when this is displayed? Is it after a signature has been saved (i.e. it means "Signature has been saved"), is it a button to use the saved signature, something else?

Reply:
It's used as a tooltip on the button used to add a saved signature:
![image](https://github.com/user-attachments/assets/8ae0d7b4-0042-4932-b873-e5e5eba5afc4)
and it'll be read by screen readers.